From a212465951e2a93837b065faa0df19f08590b816 Mon Sep 17 00:00:00 2001
From: allen yan <yanwei@marvell.com>
Date: Thu, 27 Apr 2017 17:00:29 +0800
Subject: [PATCH 0990/1345] dts: a3700: rename devel-board dts files

commit  d4ddfbb83a08ee00a314406d09bb3c9473afb376 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- This patch changes a3700 DB dts files name:
Signed-off-by: Meng Li <Meng.Li@windriver.com>
  armada-3720-db-sgmii1.dts ---> armada-3720-db-B.dts
  armada-3720-db-sfp.dts ---> armada-3720-db-C.dts
- Suffix "B" dts configurates comphy1 with sgmii1.
  Suffix "C" dts configurates comphy1 with sgmii1-sfp2.5g.

Change-Id: Ic4918d965cd7b80110c250250b59cece548ef299
Signed-off-by: allen yan <yanwei@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/38971
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
---
 arch/arm64/boot/dts/marvell/Makefile               |    4 +-
 arch/arm64/boot/dts/marvell/armada-3720-db-B.dts   |   70 +++++++++++++++++++
 arch/arm64/boot/dts/marvell/armada-3720-db-C.dts   |   73 ++++++++++++++++++++
 arch/arm64/boot/dts/marvell/armada-3720-db-sfp.dts |   73 --------------------
 .../boot/dts/marvell/armada-3720-db-sgmii1.dts     |   70 -------------------
 5 files changed, 145 insertions(+), 145 deletions(-)
 create mode 100644 arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
 create mode 100644 arch/arm64/boot/dts/marvell/armada-3720-db-C.dts
 delete mode 100644 arch/arm64/boot/dts/marvell/armada-3720-db-sfp.dts
 delete mode 100644 arch/arm64/boot/dts/marvell/armada-3720-db-sgmii1.dts

diff --git a/arch/arm64/boot/dts/marvell/Makefile b/arch/arm64/boot/dts/marvell/Makefile
index 06d3fa9..3bbf1d7 100644
--- a/arch/arm64/boot/dts/marvell/Makefile
+++ b/arch/arm64/boot/dts/marvell/Makefile
@@ -4,8 +4,8 @@ dtb-$(CONFIG_ARCH_BERLIN) += berlin4ct-stb.dtb
 
 # Mvebu SoC Family
 dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db.dtb
-dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db-sgmii1.dtb
-dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db-sfp.dtb
+dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db-B.dtb
+dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db-C.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-community.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-37x0-customer.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-espressobin.dtb
diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts b/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
new file mode 100644
index 0000000..75b5013
--- /dev/null
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
@@ -0,0 +1,70 @@
+/*
+ * Device Tree file for Marvell Armada 3720 development board
+ * (DB-88F3720-DDR3)
+ * Copyright (C) 2016 Marvell
+ *
+ * Ken Ma <make@marvell.com>
+ *
+ * This file is dual-licensed: you can use it either under the terms
+ * of the GPL or the X11 license, at your option. Note that this dual
+ * licensing only applies to this file, and not this project as a
+ * whole.
+ *
+ *  a) This file is free software; you can redistribute it and/or
+ *     modify it under the terms of the GNU General Public License as
+ *     published by the Free Software Foundation; either version 2 of the
+ *     License, or (at your option) any later version.
+ *
+ *     This file is distributed in the hope that it will be useful
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ * Or, alternatively
+ *
+ *  b) Permission is hereby granted, free of charge, to any person
+ *     obtaining a copy of this software and associated documentation
+ *     files (the "Software"), to deal in the Software without
+ *     restriction, including without limitation the rights to use
+ *     copy, modify, merge, publish, distribute, sublicense, and/or
+ *     sell copies of the Software, and to permit persons to whom the
+ *     Software is furnished to do so, subject to the following
+ *     conditions:
+ *
+ *     The above copyright notice and this permission notice shall be
+ *     included in all copies or substantial portions of the Software.
+ *
+ *     THE SOFTWARE IS PROVIDED , WITHOUT WARRANTY OF ANY KIND
+ *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY
+ *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *     OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "armada-3720-db.dts"
+/* This file enables eth1 to work on SERDES SGMII1 interface and
+ * use SMI to do PHY polling, meanwhile eth0 still works on RGMII mode.
+ */
+/ {
+	soc {
+		internal-regs {
+			eth1: ethernet@40000 {
+				phy-mode = "sgmii";
+				phy = <&phy1>;
+				phys = <&a3700_comphy 1 COMPHY_SGMII1>;
+				phy-names = "comphy";
+				status = "okay";
+				buffer-manager = <&bm>;
+				bm,pool-long = <2>;
+				bm,pool-short = <3>;
+			};
+			usb3@58000 {
+				status = "disabled";
+			};
+		};
+	};
+};
+
diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db-C.dts b/arch/arm64/boot/dts/marvell/armada-3720-db-C.dts
new file mode 100644
index 0000000..1adfb2a
--- /dev/null
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db-C.dts
@@ -0,0 +1,73 @@
+/*
+ * Device Tree file for Marvell Armada 3720 development board
+ * (DB-88F3720-DDR3)
+ * Copyright (C) 2016 Marvell
+ *
+ * Ken Ma <make@marvell.com>
+ *
+ * This file is dual-licensed: you can use it either under the terms
+ * of the GPL or the X11 license, at your option. Note that this dual
+ * licensing only applies to this file, and not this project as a
+ * whole.
+ *
+ *  a) This file is free software; you can redistribute it and/or
+ *     modify it under the terms of the GNU General Public License as
+ *     published by the Free Software Foundation; either version 2 of the
+ *     License, or (at your option) any later version.
+ *
+ *     This file is distributed in the hope that it will be useful
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ * Or, alternatively
+ *
+ *  b) Permission is hereby granted, free of charge, to any person
+ *     obtaining a copy of this software and associated documentation
+ *     files (the "Software"), to deal in the Software without
+ *     restriction, including without limitation the rights to use
+ *     copy, modify, merge, publish, distribute, sublicense, and/or
+ *     sell copies of the Software, and to permit persons to whom the
+ *     Software is furnished to do so, subject to the following
+ *     conditions:
+ *
+ *     The above copyright notice and this permission notice shall be
+ *     included in all copies or substantial portions of the Software.
+ *
+ *     THE SOFTWARE IS PROVIDED , WITHOUT WARRANTY OF ANY KIND
+ *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY
+ *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *     OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "armada-3720-db.dts"
+/* This file enables eth1 to work in SGMII mode with 1g or 2.5g fixed-link,
+ * eth0 still works in RGMII mode.
+ */
+/ {
+	soc {
+		internal-regs {
+			eth1: ethernet@40000 {
+				phy-mode = "sgmii";
+				status = "okay";
+				fixed-link {
+					/* Here "speed" is set to 1000, GBE MAC is running in 1G mode, */
+					/* but the actual MAC speed may be 1 Gbps or 2.5 Gbps, */
+					/* it's up to the corresponding SERDES speed in u-boot fdt comphy node. */
+					/* If SERDES speed is set to 3.125G in u-boot, it implies sgmii 2.5 Gbps; */
+					/* if SERDES speed is set to 1.25G in u-boot, it implies sgmii 1 Gbps. */
+					speed = <1000>;
+					full-duplex;
+				};
+			};
+			usb3@58000 {
+				status = "disabled";
+			};
+		};
+	};
+};
+
diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db-sfp.dts b/arch/arm64/boot/dts/marvell/armada-3720-db-sfp.dts
deleted file mode 100644
index 1adfb2a..0000000
--- a/arch/arm64/boot/dts/marvell/armada-3720-db-sfp.dts
+++ /dev/null
@@ -1,73 +0,0 @@
-/*
- * Device Tree file for Marvell Armada 3720 development board
- * (DB-88F3720-DDR3)
- * Copyright (C) 2016 Marvell
- *
- * Ken Ma <make@marvell.com>
- *
- * This file is dual-licensed: you can use it either under the terms
- * of the GPL or the X11 license, at your option. Note that this dual
- * licensing only applies to this file, and not this project as a
- * whole.
- *
- *  a) This file is free software; you can redistribute it and/or
- *     modify it under the terms of the GNU General Public License as
- *     published by the Free Software Foundation; either version 2 of the
- *     License, or (at your option) any later version.
- *
- *     This file is distributed in the hope that it will be useful
- *     but WITHOUT ANY WARRANTY; without even the implied warranty of
- *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *     GNU General Public License for more details.
- *
- * Or, alternatively
- *
- *  b) Permission is hereby granted, free of charge, to any person
- *     obtaining a copy of this software and associated documentation
- *     files (the "Software"), to deal in the Software without
- *     restriction, including without limitation the rights to use
- *     copy, modify, merge, publish, distribute, sublicense, and/or
- *     sell copies of the Software, and to permit persons to whom the
- *     Software is furnished to do so, subject to the following
- *     conditions:
- *
- *     The above copyright notice and this permission notice shall be
- *     included in all copies or substantial portions of the Software.
- *
- *     THE SOFTWARE IS PROVIDED , WITHOUT WARRANTY OF ANY KIND
- *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
- *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
- *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
- *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY
- *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
- *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
- *     OTHER DEALINGS IN THE SOFTWARE.
- */
-
-#include "armada-3720-db.dts"
-/* This file enables eth1 to work in SGMII mode with 1g or 2.5g fixed-link,
- * eth0 still works in RGMII mode.
- */
-/ {
-	soc {
-		internal-regs {
-			eth1: ethernet@40000 {
-				phy-mode = "sgmii";
-				status = "okay";
-				fixed-link {
-					/* Here "speed" is set to 1000, GBE MAC is running in 1G mode, */
-					/* but the actual MAC speed may be 1 Gbps or 2.5 Gbps, */
-					/* it's up to the corresponding SERDES speed in u-boot fdt comphy node. */
-					/* If SERDES speed is set to 3.125G in u-boot, it implies sgmii 2.5 Gbps; */
-					/* if SERDES speed is set to 1.25G in u-boot, it implies sgmii 1 Gbps. */
-					speed = <1000>;
-					full-duplex;
-				};
-			};
-			usb3@58000 {
-				status = "disabled";
-			};
-		};
-	};
-};
-
diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db-sgmii1.dts b/arch/arm64/boot/dts/marvell/armada-3720-db-sgmii1.dts
deleted file mode 100644
index 75b5013..0000000
--- a/arch/arm64/boot/dts/marvell/armada-3720-db-sgmii1.dts
+++ /dev/null
@@ -1,70 +0,0 @@
-/*
- * Device Tree file for Marvell Armada 3720 development board
- * (DB-88F3720-DDR3)
- * Copyright (C) 2016 Marvell
- *
- * Ken Ma <make@marvell.com>
- *
- * This file is dual-licensed: you can use it either under the terms
- * of the GPL or the X11 license, at your option. Note that this dual
- * licensing only applies to this file, and not this project as a
- * whole.
- *
- *  a) This file is free software; you can redistribute it and/or
- *     modify it under the terms of the GNU General Public License as
- *     published by the Free Software Foundation; either version 2 of the
- *     License, or (at your option) any later version.
- *
- *     This file is distributed in the hope that it will be useful
- *     but WITHOUT ANY WARRANTY; without even the implied warranty of
- *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *     GNU General Public License for more details.
- *
- * Or, alternatively
- *
- *  b) Permission is hereby granted, free of charge, to any person
- *     obtaining a copy of this software and associated documentation
- *     files (the "Software"), to deal in the Software without
- *     restriction, including without limitation the rights to use
- *     copy, modify, merge, publish, distribute, sublicense, and/or
- *     sell copies of the Software, and to permit persons to whom the
- *     Software is furnished to do so, subject to the following
- *     conditions:
- *
- *     The above copyright notice and this permission notice shall be
- *     included in all copies or substantial portions of the Software.
- *
- *     THE SOFTWARE IS PROVIDED , WITHOUT WARRANTY OF ANY KIND
- *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
- *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
- *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
- *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY
- *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
- *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
- *     OTHER DEALINGS IN THE SOFTWARE.
- */
-
-#include "armada-3720-db.dts"
-/* This file enables eth1 to work on SERDES SGMII1 interface and
- * use SMI to do PHY polling, meanwhile eth0 still works on RGMII mode.
- */
-/ {
-	soc {
-		internal-regs {
-			eth1: ethernet@40000 {
-				phy-mode = "sgmii";
-				phy = <&phy1>;
-				phys = <&a3700_comphy 1 COMPHY_SGMII1>;
-				phy-names = "comphy";
-				status = "okay";
-				buffer-manager = <&bm>;
-				bm,pool-long = <2>;
-				bm,pool-short = <3>;
-			};
-			usb3@58000 {
-				status = "disabled";
-			};
-		};
-	};
-};
-
-- 
1.7.9.5

